source/validate.cpp

```diff
@@ -341,6 +333,14 @@ spv_result_t ValidateBinaryUsingContextAndValidationState(
                         context.opcode_table, context.operand_table,
                         context.ext_inst_table, *vstate, &position);
 }
+}  // anonymous namespace
+
+spv_result_t spvValidate(const spv_const_context context,
+                         const spv_const_binary binary,
+                         spv_diagnostic* pDiagnostic) {
+  return spvValidateBinary(context, binary->code, binary->wordCount,
+                           pDiagnostic);
+}
 
```

Comment:
Sorry if I'm missing something obvious.  But it looks like the function is still out of the anonymous namespace.

Reply:
I wasn't specific.  :-)

ValidateBinaryUsingContextAndValidationState was outside the anonymous namespace.  I moved spvValidate down and extended the anonymous namespace region down to enclose ValidateBinaryUsingContextAndValidationState.

Reply:
spvValidate is part of our public extern "C" API, and should remain there.

Reply:
Ah, ok.  I figured I was missing something :)

Thanks.